java/src/test/java/org/rocksdb/DBOptionsTest.java

```diff
@@ -8,8 +8,8 @@
 import org.junit.ClassRule;
 import org.junit.Test;
 
-import java.util.Properties;
-import java.util.Random;
+import java.nio.file.Paths;
+import java.util.*;
```

Comment:
Better to spell out individual classes instead of a * (unless there are too many, of course).